### ISSUE TITLE
Clean up code

### ISSUE THREAD

**Issue by Sammidysam** [frc461/website-2013#66]
We should clean up this code so that it is easier to modify.  We can let Emacs do most of the work.  Is this the configuration for HTML indentation that we want?

``` lisp
(add-hook 'html-mode-hook
          (lambda ()
            (setq sgml-basic-offset 4
                  indent-tabs-mode t)))
```

And, of course, running RHTML mode while indenting.  Indentation will then work perfectly unless we run into [this issue](https://github.com/eschulte/rhtml/issues/10).
###### Steps that need to be taken
- Remove internationalization things.
  
  In `app/views/events/new.html.erb`, these are `<%- model_class = Event -%>` and `<%=t '.title', :default => t('helpers.titles.new', :model => model_class.model_name.human, :default => "New #{model_class.model_name.human}") %>`.
- Use title method instead of `<% content_for :title do "New Event" end %>` in files.
- Re-indent all of the code.
- Become consistent with single or double quotes.
- Add newlines where they help.

There are probably more items that we need to do as well, but these will help a lot.

**Comment by Sammidysam** [comment]
We should use the branch `re-indent` for this, even though it is more than re-indenting.

**Comment by Sammidysam** [comment]
I have deleted that branch because it had a bad name, and replaced it with a new branch, `clean-up`, which should be used for cleaning up code.

**Comment by Sammidysam** [comment]
@four04 What is so bad about a hyphen in the branch name that causes it to be necessary to delete a branch containing 20 commits (there were this many, right?) that had not been merged to master?  Or did you merge them?  If not, I have them at my laptop, so I can sort this out, but I'm curious of your thought process.

**Comment by rye** [comment]
There were like 80 merge conflicts so I assumed that the branch was poorly-maintained and not currently in use. I always want branches to be conflicting as little as possible, and so that probably could have been avoided if all of the changes on master were merged into the `clean-up` branch. If you want to spend some time resolving merge conflicts and re-adding those commits, that'd be great.

**Comment by Sammidysam** [comment]
I am going to do that.  I'll delete the `cleanup` branch once it has been merged to `clean-up`.  I'll only push stuff when everything is done.  The merge conflicts would still be present if we merged `master` into `clean-up` each time we committed; the real culprit is that some commits to master were doing cleaning activities, namely https://github.com/team461WBI/website/commit/10620cd7ccd6f399341f633ef013701033efb854.

**Comment by cg505** [comment]
Yeah I kinda forgot/didn't know about this branch.

**Comment by samontea** [comment]
:+1:

**Comment by rye** [comment]
These two branches should be "compatible." That is, we should be able to merge between them with very few conflicts. Content changes (changes not in whitespace) should be done on `master` or some other branch, and whitespace changes on `clean-up`. These two types of changes are not conflicting, and, as such, our branching model should probably accommodate merging between both branches (though we should still keep it to a minimum and just merge `master` into `clean-up` until we're more done.

**Comment by Sammidysam** [comment]
Yep, that is the plan.

**Comment by rye** [comment]
Okay. Do you have an ETA for the merge of the off-line `clean-up` branch into the `cleanup` branch?

**Comment by Sammidysam** [comment]
The initial merge is done, but some things still need to be changed because I obliterated a few things.  Since school has started again, who knows when it will be done.  My guess is that it will be done the next time that I go to a Robotics meeting.

**Comment by rye** [comment]
By the way: since you weren't there for clean-up yesterday (You didn't need to be, but I was), we don't have any meetings this week.

**Comment by Sammidysam** [comment]
Yeah, that slowed down this getting done.  I _highly_ doubt I will work on this without being at Robotics.

**Comment by Sammidysam** [comment]
It is now done.  We need to thoroughly check it for errors, though.

**Comment by rye** [comment]
Have you checked and seen if it throws any errors under normal usage? I'll
go through and read the diff later this evening.

**Comment by Sammidysam** [comment]
Yes.  https://github.com/team461WBI/website/commit/7c0c7021f8e4b255f2cee3eb83269ebe317a1c55 tried to fix that.

**Comment by Sammidysam** [comment]
Have you deemed the current state of `clean-up` as stable?  If so, it might be good to merge it to `master`.

**Comment by rye** [comment]
I'll run some tests on it and get back to you soon.

**Comment by rye** [comment]
With my standard usage test, nothing died, so I'll merge it now.

**Comment by rye** [comment]
This issue will be used for conversation regarding refactoring as well, as it is a form of cleanup.

**Comment by rye** [comment]
Partial work on this has been merged to `master` as of dea94422c2a8aa6fd8faf8a5fedf2f8d54bb01a8. Further work will probably be done eventually on the `clean-up` branch.

**Comment by Sammidysam** [comment]
There is no need to do this anymore since we are making a new website.
